source/postprocess/visualization/dynamic_topography.cc

```diff
@@ -207,7 +207,15 @@ namespace aspect
                                  "in the outputted data file (not visualization). "
                                  "'true' subtracts the mean, 'false' leaves "
                                  "the calculated dynamic topography as is. ");
-
+              prm.declare_entry ("Density above","0",
+                                 Patterns::Double (0),
+                                 "Dynamic topography is calculated as the excess or lack of mass that is supported by mantle flow. "
+                                 "This value depends on the density of material that is moved up or down, i.e. crustal rock, and the "
+                                 "density of the material that is displaced (generally water or air). While the density of crustal rock "
+                                 "is part of the material model, this parameter 'density_above' allows the user to specify the density "
```

Comment:
Hi Shangxin,
last minor thing: I meant to replace 'density_above' by 'Density above' (the name of the input parameter, with space instead of underscore) so that users can directly see what to change in the input file. Maybe the formatting was lost on Github.